third-party/rspamd/docker-compose-rspamd-with-kvrocks-sentinel.yml

```diff
@@ -103,7 +103,7 @@ services:
       kvrocks-master:
           condition: service_healthy
     container_name: rspamd
-    image: rspamd/rspamd:3.9.1
+    image: rspamd/rspamd:3.11.1 # Rspamd version >= 3.10 is needed to fix the Redis read-only issue when Rspamd gets bayes statistics too.
```

Comment:
Maybe add a section about KvRocks in the README.md of the rspamd third party module? 

Presenting quickly about kvrocks and why it could be a better choice than redis if you have per suer bayes turned on, but also precising this kind of things that rspamd version needs to be >= 3.10?

Reply:
Done.